apps/test/util/testUtils.js

```diff
@@ -342,6 +342,12 @@ export function restoreOnWindow(key) {
   delete originalWindowValues[key];
 }
 
+export function sandboxDocumentBody() {
```

Comment:
Worth a comment explaining what it does, why it does it, and what sorts of drawbacks in might have (i.e. the fact that this will blow away attached event handlers).